### DISCUSSION TITLE
How to Check Workflow each Experiment Status

### DISCUSSION THREAD

**Discussion by Jim0530** [chaos-mesh/chaos-mesh#3916] [Q&A]
## Feature Request

**Is your feature request related to a problem? Please describe:**

I'm always frustrated when controlling the timestamp of experiment status (start & end & recover), i want to integrate a workflow for chaos experiment and data crawling (using task to call api) to integrate with my CI/CD pipeline.

**Describe the feature you'd like:**

When triggering workflow, there's a deadline to control the duration time of the experiment. The experiment info will be deleted as soon as the deadline exceeded. i am wondering how can i check each experiment status (ALL recovered, ALL injected...)

**Comment by STRRL** [comment]
You could take a look on the resource `WorkflowNode`, there are some status about the experiment.

**Reply by Jim0530** [reply]
Thanks a lot! But when i describe workflownode there's only a few information, which can't tell whether the experiment is successful or not???
